README.md

```diff
@@ -1,10 +1,70 @@
 # disblob
+A stylesheet(css) generator for use with Discord.
+***
+***All original code is done by the insane but genius [diamondburned](https://github.com/diamondburned/)***
+> [Master repo](https://github.com/diamondburned/disblob)
 
-A stylesheet generator for use with Discord.
+***This fork is an attempt to refine the `README` and make it easier for others to setup!***
+***
+# Status
+|   |Status |	|
+|--|--|--|
+|   |Edits: |	|
+|  | `README` Only |  |
 
+|   |Fork|  |
+|--|--|--|
+|  | Fork Only |  |
+|  |  (PR [Open](https://github.com/diamondburned/disblob/pull/1))  |  |
+
+|  | Testing |  |
+|--|--|--|
+|  | 		-Client + Firefox-		 |  |
+|  |-  (Tests have been done with the client and Firefox)  |  
+***
+ - [x] Client (BD(BetterBiscord)~(100% Working!))
+ - [ ] Client (Other css injectors)
+ - [ ] Browsers (Chrome, etc. (Use [stylus](https://github.com/openstyles/stylus) to inject css)
+ - [x] Firefox Browser (Testing done with stylish(DO NOT USE(Considered spyware)) 
+***
+***
+|  | Contributors:|  |
+|--|--|--|
+|  | 		2 ([diamondburned](https://github.com/diamondburned/) & [ThatGeekyWeeb](https://github.com/ThatGeekyWeeb))
+
+***
+## Dependencies
+  1. (All Blobmoji deps(https://github.com/C1710/blobmoji/wiki/Build-instructions))
+  2. `go`
+  3. `zopfli`
+  4. 1+ GiB of storage space
```

Comment:
This is probably not needed either, but it should be obvious anyway.

Reply:
Moved to blobmoji deps, and removed the storage warning